change/@fluentui-react-button-aaf5c99d-c063-4562-9ec1-42c216a4b2d7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31700&commitId=9c14726e6b2cc13ba33afef3ac2707de9efe6599&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Added|
|Card Converged.appearance - RTL.normal.chromium.png|0| Added|
|Card Converged.orientation.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>